README.md

```diff
@@ -260,41 +267,43 @@ key is set to
-representation of the relevant `GTMAppAuthFetcherAuthorization` instance is supplied as the value for
+representation of the relevant `GTMAuthSession` instance is supplied as the value for
 [`kSecValueData`](https://developer.apple.com/documentation/security/ksecvaluedata?language=objc)
 and this is encrypted and stored by
 [Keychain Services](https://developer.apple.com/documentation/security/keychain_services?language=objc).
 
 For macOS, two Keychain storage options are available: the traditional file-based Keychain storage
 which uses access control lists and the more modern [data protection keychain storage](https://developer.apple.com/documentation/security/ksecusedataprotectionkeychain?language=objc)
-which uses Keychain access control groups. By default, GTMAppAuth uses the file-based Keychain storage on macOS.  You may opt into using data protection keychain storage by using the parameter
-`useDataProtectionKeychain:YES` in your method calls.  Note that Keychain items stored via one
-storage type will not be available via the other and macOS apps that choose to use the data
-protection Keychain will need to be signed in order for Keychain operations to succeed.
+which uses Keychain access control groups. By default, GTMAppAuth uses the file-based Keychain storage on macOS.  You may opt
+into using data protection keychain storage by including the `GTMKeychainAttribute.useDataProtectionKeychain` attribute
+in the `keychainAttributes` parameter of `initWithItemName:keychainAttributes:` when initializing `GTMKeychainStore`.
+Note that Keychain items stored via one storage type will not be available via the other and macOS apps that choose
+to use the data protection Keychain will need to be signed in order for Keychain operations to succeed.
 
 #### GTMOAuth2 Compatibility
 
-To assist the migration from GTMOAuth2 to GTMAppAuth, GTMOAuth2-compatible
-Keychain methods are provided in `GTMOAuth2KeychainCompatibility`.
+To assist the migration from GTMOAuth2 to GTMAppAuth, GTMOAuth2-compatible Keychain methods are provided in `GTMKeychainStore`.
 
 ```objc
-// Restore from Keychain
-GTMAppAuthFetcherAuthorization *auth =
-    [GTMOAuth2KeychainCompatibility authForGoogleFromKeychainForName:kKeychainItemName
-                                                            clientID:clientID
-                                                        clientSecret:clientSecret];
+GTMKeychainStore keychainStore = [[GTMKeychainStore alloc] initWithItemName:kKeychainItemName];
 
-// Remove from Keychain
-[GTMOAuth2KeychainCompatibility removeAuthFromKeychainForName:kKeychainItemName];
+// Retrieve from the Keychain
+NSError *error;
+GTMAuthSession *authSession =
+    [keychainStore retrieveAuthSessionForGoogleInGTMOAuth2FormatWithClientID:clientID
+                                                                clientSecret:clientSecret
+                                                                error:&error];
```

Comment:
Formatting: align `error:` with the other parameters.

Reply:
done